organizers/2018-04-10.html

```diff
@@ -0,0 +1,145 @@
+    </p>
+</section>
+
+<section style="text-align: left">
+    <h1>📋 Today's Schedule</h1>
+    <h2>🎺 Django News</h2>
+</section>
+
+<section style="text-align: left">
+    <h1 style="text-align: center;">📋 Today's Schedule</h1>
+    <h2 style="text-align: center;">Talks</h2>
+    <ol>
+        <li style="overflow: hidden;">
+            <img src="https://res.cloudinary.com/skillsmatter/image/upload/c_crop,g_custom/c_scale,w_96,h_96/v1522945378/jf2mwcxlcurnze6iieyt.png"
+                style="float:left; border-radius: 100%; width: 150px; height 150px; margin: 5px 50px;"> Adam Charnock
+                - Freelancing (and Contracting) for Developers
+        </li>
+        <li style="overflow: hidden;">
+            <img src="https://res.cloudinary.com/skillsmatter/image/upload/c_crop,g_custom/c_scale,w_96,h_96/v1488977535/scrxd00x1vvwijoildba.png"
+                style="float:left; border-radius: 100%; width: 150px; height 150px; margin: 5px 50px;"> Dan Palmer
+                - django-enumfield
+        </li>
+    </ol>
+</section>
+
+<section style="text-align: left">
+    <h1>📋 Today's Schedule</h1>
+    <h2>Eating and Drinking</h2>
+    <ol>
+        <li><strong>Until ~20:45</strong> - Drinks and Pizza at<br>The Space Bar</li>
+        <li><strong>20:45 - close</strong>: Red Lion Pub (opposite)</li>
+    </ol>
+</section>
+
+<section>
+    <h1>🎺 Django News</h1>
+    <h2>Security Release</h2>
+    <ul>
+        <li>2.0.4 and 1.11.12 released last Monday</li>
+        <li>Bugfix release</li>
+        <li>Fixes crash when combining two annotated <code>values_list()</code> querysets in some circumstances.</li>
+        <li><a href="https://www.djangoproject.com/weblog/">https://www.djangoproject.com/weblog/</a></li>
+    </ul>
+</section>
+
+<section>
+    <h1>🎺 Django News</h1>
+    <h2>DjangoCon Europe</h2>
+    <ul>
+        <li>Speakers announced!</li>
+        <li>Heidelberg 🇩🇪</li>
+        <li>23&ndash;27 May</li>
+        <ul>
+            <li>23rd&ndash;25th: Main Conference</li>
+            <li>26th &amp; 27th: Sprints &amp; Workshops</li>
+        </ul>
+        <li>Remember: 28th is a bank holiday in the UK!</li>
+        <li><a href="https://2018.djangocon.eu/">https://2018.djangocon.eu/</a></li>
+    </ul>
+</section>
+
+<section>
+    <h1>🎺 Django News</h1>
+    <h2>EuroPython</h2>
+    <ul>
+        <li>Edinburgh 🏴󠁧󠁢󠁳󠁣󠁴󠁿</li>
+        <li>23&ndash;29 July</li>
+        <ul>
+            <li>23rd &amp; 24th: Workshops and Training</li>
+            <li>25th&ndash;27th: Main Conference</li>
+            <li>28th &amp; 29th: Sprints (Hackathon)</li>
+        </ul>
+        <li><a href="https://ep2018.europython.eu/">https://ep2018.europython.eu/</a></li>
+    </ul>
+</section>
+
+<section>
```

Comment:
Oh, we were discussing having a slide just before the talks start, or after, asking for people to approach us for talks. Could you add one?